statistics/table.go

```diff
@@ -406,7 +407,18 @@ func (coll *HistColl) getEqualCondSelectivity(idx *Index, bytes []byte, coverAll
 			// for equality queries
 			return float64(coll.ModifyCount) / float64(idx.NDV) / idx.TotalRowCount()
 		}
-		// for range queries
+		// The equal condition only uses prefix columns of the index.
+		colIDs := coll.Idx2ColumnIDs[idx.ID]
+		var ndv int64
+		for i, colID := range colIDs {
+			if i >= usedColsLen {
+				break
+			}
+			ndv = mathutil.MaxInt64(ndv, coll.Columns[colID].NDV)
```

Comment:
Why not use the `col1.ndv * col2.ndv * ...` instead of max of them?

Reply:
If the 2 rows are like (1,1),(2,2), the NDV if 2. Also, using `max` is consistent with `getCardinality`.